### DISCUSSION TITLE
"NASCAR" stands for National Association for "What" Car Auto Racing?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#599] [Galaxy Trivia] [answered]
**Bonus:** According to the official rules, how many numbers must each car display?

**Comment by peterbaccaro** [comment]
NASCAR stands for National Association for **Stock** Car Auto Racing.

According to the official rules of NASCAR, each car must display **three** numbers in a specific manner. 

1) Car Number: This is the primary number that identifies the car and is displayed on the doors of the car. It must be at least 18 inches tall and must be clearly visible from both the driver's and passenger's side of the car.

2) Roof Number: This is the number that is displayed on the roof of the car and is used for scoring purposes. It must be at least 4 inches tall and must be visible from overhead.

3) Rear Window Number: This is the number that is displayed on the rear window of the car and is used for identification during races. It must be at least 4 inches tall and must be clearly visible from the rear of the car.

These numbers are required to be clearly displayed on the car at all times during NASCAR races and must adhere to the specific size and placement requirements as outlined in the NASCAR rulebook.

**Comment by thuongtruong1009** [comment, marked as answer]
"NASCAR" stands for National Association for Stock Car Auto Racing.

According to the official rules, each car in a NASCAR race must display three numbers: one on the doors, one on the roof, and one on the rear panel. These numbers help identify the car for officials and fans during the race.

**Comment by carlostipton1319** [comment]
NASCAR race must display three numbers: one on the doors, one on the roof, and one on the [reasons ](https://beautyhomedesigns.com/top-5-reasons-to-hire-a-professional-for-your-garage-door-repair-needs/)panel.this way you can get the best beautty
